### PR TITLE
Report errors when running npm run update [Fix #568]

### LINKED ISSUE #568: Report errors on all landscapes when running daily update
We don't have any visibility on any failures produced when running the daily `npm run update` to refresh the data for landscapes other than CNCF.

In fact, I added an the `Data Refreshed` column on [landscapes-dev.netlify.app](https://landscapes-dev.netlify.app), that should show the latest time this task was run on a landscape. It's not showing on three of the landscapes, which probably means the task fails for these landscapes.

We should report any failures somewhere, I see a couple of options, either post them on the `landscapes-logs` Slack channel or open an issue on the relevant github repo.

### PR DESCRIPTION
**Pull request by jordinl83**

### CODE REVIEW

**Comment by jordinl83** [comment]
@AndreyKozlov1984 please review and merge this. Code to send update logs to new slack channel and some formatting for the message

File: update_server/setup.bash [hunk 1]
```diff
@@ -21,6 +21,7 @@ export CRUNCHBASE_KEY=key
 export GITHUB_TOKEN=token
 export GITHUB_USER=CNCF-Bot
 export GITHUB_KEY=key
+export SLACK_CHANNEL=channel
```
Comment (AndreyKozlov1984) [inline update_server/setup.bash:24]: SLACK_ERROR_CHANNEL

**Comment by AndreyKozlov1984** [comment]
Have you tried to test this somehow ?

**Comment by jordinl83** [comment]
@AndreyKozlov1984 Haven't tried the full `landscapes.sh` but I've tried the parts responsible for posting on the channel and they seem to work.

**Comment by jordinl83** [comment]
@AndreyKozlov1984 this should be ready

**Pull request merged**